### PR TITLE
Fix description field

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -242,5 +242,5 @@
   "downloadCases": "Download Case(s)",
 
   "descriptionModificationError": "An error when modifying the description",
-  "descriptionModificationDialog": "Description modification"
```
Comment (AbdelHedhili) [inline src/translations/en.json:269]: this wasn't used ?

**Review by AAJELLAL — CHANGES_REQUESTED** [review]
- The application crashes when creating a study or situation regardless of adding the description or not.
- The description counter needs some margin at the bottom.

**Review by AAJELLAL — APPROVED** [review]
Tests & code OK.

**Review by AbdelHedhili — APPROVED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (YenguiSeddik) [reply src/translations/en.json:269]: No, it was used in the description modification dialog but was replaced by "description": "Description",

**Review by souissimai — APPROVED** [review]
test ok

**Pull request merged**